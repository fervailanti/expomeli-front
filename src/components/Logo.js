import React from 'react'
import { images } from '../images'

const Logo = ({ type = 'lite', size = 'medium', onClick }) => {
  const variants = {
    full: images.mercadolibreFull,
    lite: images.mercadolibreReduced
  }
  const gauges = {
    small: 'h-10',
    medium: 'h-12',
    large: 'h-16'
  }
  return (
    <img
      src={variants[type]}
      className={`${gauges[size]} cursor-pointer`}
      alt="mercadolibre_logo"
      onClick={onClick}
    />
  )
}

export default Logo
