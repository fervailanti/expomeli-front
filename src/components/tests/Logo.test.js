import React from 'react'
import Logo from '../Logo'
import { render } from '@testing-library/react'

it('Logo component should render correctly', () => {
  const tree = render(<Logo />)
  expect(tree.container).toMatchSnapshot()
})
