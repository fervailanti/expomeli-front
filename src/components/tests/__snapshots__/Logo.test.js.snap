// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo component should render correctly 1`] = `
<div>
  <img
    alt="mercadolibre_logo"
    class="h-12 cursor-pointer"
    src="mercadolibre_logo_reduced.png"
  />
</div>
`;
